src/main/java/org/openrewrite/java/template/processor/TemplateDescriptor.java

```diff
@@ -148,7 +148,15 @@ public String toJavaTemplateBuilder(int pos) {
         }
 
         List<JCTree.JCTypeParameter> typeParameters = classDecl.typarams == null ? emptyList() : classDecl.typarams;
-        return TemplateCode.process(tree, method.getReturnType().type, method.getParameters(), typeParameters, pos, method.restype.type instanceof Type.JCVoidType, true);
+        return TemplateCode.process(
+                tree,
+                method.getReturnType().type,
+                method.getParameters(),
+                typeParameters,
+                pos,
+                method.restype.type instanceof Type.JCVoidType,
+                true,
+                true);
```

Comment:
We could add an option here to make it opt out/in to use classpathFromResources, much like folks can switch between javax/jakarta generated annotations.